docs/podman-remote.1.md

```diff
@@ -0,0 +1,139 @@
+% podman-remote(1)
+
+## NAME
+podman - Simple management tool for pods, containers and images
```

Comment:
suggest:
`podman-remote - A remote  CLI for Podman; a simple management tool  for pods, containers and images`

@mheon WDYT

Reply:
Sure, works for me

Reply:
SGTM